### PR TITLE
IDEM-246

### PR DESCRIPTION
**Pull request by JasminFarhi**
Screen Shot 2022-07-15 at 1 27 12 pm

### CODE REVIEW

File: src/components/form.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+const RedeemCodeForm = () => {
+  const { register, handleSubmit } = useForm<Inputs>();
+  const onSubmit: SubmitHandler<Inputs> = (data) => {
+    axios
+      .post(`https://proxy.idem.com.au/registerTester`, { data })
+      .then((res) => {
+        console.log(res);
+        console.log(res.data);
+        alert(
+          "We have submitted your request. Please check your email for a confirmation."
```
Comment (bitcoinbrisbane) [inline src/components/form.tsx]:
```suggestion
          "We have submitted your request. Please check your email for a redeem code."
```

File: src/components/form.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        </div>
+        <form onSubmit={handleSubmit(onSubmit)} className="flex-col">
+          <div>
+            <label htmlFor="firstName" className="text-white text-xl">
+              First Name
+            </label>
+            <div>
+              <input
+                className=" outline-[#ffd000] rounded-3xl text-white bg-black border-white border-2 ml-5 mt-2 px-2 py-2 text-sm w-80"
+                placeholder="Write here..."
```
Comment (bitcoinbrisbane) [inline src/components/form.tsx]: Remove

File: src/components/form.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            </div>
+          </div>
+
+          <div>
+            <label htmlFor="lastName" className="text-white text-xl">
+              Last Name
+            </label>
+            <div>
+              <input
+                placeholder="Write here..."
```
Comment (bitcoinbrisbane) [inline src/components/form.tsx]: Remove

**Review by bitcoinbrisbane — DISMISSED** [review]
Few things to look at.  Remove the empty files too please

File: src/components/form.tsx (hunk 2 above)
Comment (JasminFarhi) [reply src/components/form.tsx]: I'm not sure what you mean to remove from this comment?

File: src/components/form.tsx (hunk 3 above)
Comment (JasminFarhi) [reply src/components/form.tsx]: If you mean the place holder, that's what's in the figma?

File: package.json [hunk 4]
```diff
@@ -13,9 +13,12 @@
     "test": "cross-env TS_NODE_PROJECT='./tsconfig.spec.json' mocha"
   },
   "dependencies": {
+    "axios": "^0.24.0",
+    "i": "^0.3.7",
```
Comment (pondpiu) [inline package.json]: what's "i" package for?

**Review by pondpiu — DISMISSED** [review]

**Review by pondpiu — CHANGES_REQUESTED** [review]
Remove package-lock.json as we're using yarn in this project
remove empty files `src/apis/Api.ts` and `src/apis/HTTPClient.ts`

File: .env.development [hunk 5]
```diff
@@ -1 +1 @@
-VITE_APP_PROXY_URL="http://proxy.idem.com.au/"
\ No newline at end of file
+VITE_APP_PROXY_URL="http://proxy.idem.com.au/"
```
Comment (bitcoinbrisbane) [inline .env.development:1]:
```suggestion
VITE_APP_PROXY_URL="https://proxy.idem.com.au/"
```

Comment (bitcoinbrisbane) [inline .env.development]:
```suggestion
VITE_APP_PROXY_URL="https://proxy.idem.com.au/"
```

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
One minor thing

File: .env.development (hunk 5 above)
Comment (JasminFarhi) [reply .env.development]: It needs to be http :)

**Review by pondpiu — DISMISSED** [review]
LGTM

**Review by pondpiu — APPROVED** [review]
LGTM

**Review by OptimusOpus — APPROVED** [review]
LGTM

**Pull request merged**